<launch>
 
  <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen" >

    <rosparam file="$(find jackal_navigation)/params/costmap_common_params.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find jackal_navigation)/params/costmap_common_params.yaml" command="load" ns="local_costmap" />
    
    <rosparam file="$(find jackal_navigation)/params/map_nav_params/local_costmap_params.yaml" command="load" />
    <rosparam file="$(find jackal_navigation)/params/map_nav_params/global_costmap_params.yaml" command="load" />

    <rosparam file="$(find jackal_navigation)/params/prohibition_areas.yaml" command="load" ns="global_costmap/costmap_prohibition_layer" />
    <rosparam file="$(find jackal_navigation)/params/prohibition_areas.yaml" command="load" ns="local_costmap/costmap_prohibition_layer" />



<!--     <rosparam file="$(find jackal_navigation)/params/dwa_local_planner_params.yaml" command="load" /> -->
    <!-- <rosparam file="$(find jackal_navigation)/params/base_local_planner_params.yaml" command="load" /> -->
    <rosparam file="$(find jackal_navigation)/params/teb_local_planner_params.yaml" command="load" />


    <rosparam file="$(find jackal_navigation)/params/costmap_converter_params.yaml" command="load" />

<!--     <rosparam file="$(find jackal_navigation)/params/global_planner_params.yaml" command="load" /> -->
<!--     <rosparam file="$(find jackal_navigation)/params/move_base_params.yaml" command="load" /> -->


    <!-- <param name="base_global_planner" value="Astar_planner/AstarPlannerROS"/> -->
    <param name="base_global_planner" value="global_planner/GlobalPlanner" />
    <param name="planner_frequency" value="1.0" />
    <!-- <param name="base_global_planner" type="string" value="navfn/NavfnROS" /> -->
    <param name="base_local_planner" value="teb_local_planner/TebLocalPlannerROS" />
    <param name="controller_frequency" value="10.0" />
<!--     <param name="base_local_planner" value="dwa_local_planner/DWAPlannerROS" /> -->
<!--     <param name="controller_frequency" value="10.0" /> -->
    <!-- <param name="base_local_planner" value="base_local_planner/TrajectoryPlannerROS"/> -->

    <remap from="odom" to="odometry/filtered" />
  </node>

</launch>
